engine/src/test/scala/cromwell/webservice/routes/wes/ServiceInfoSpec.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.webservice.routes.wes
+
+import akka.actor.Props
+import akka.http.scaladsl.testkit.ScalatestRouteTest
+import akka.util.Timeout
+import cromwell.engine.language.{CromwellLanguages, LanguageConfiguration}
+import cromwell.webservice.routes.CromwellApiService
+import cromwell.webservice.routes.CromwellApiServiceSpec.MockWorkflowStoreActor
+import org.scalatest.{AsyncFlatSpec, Matchers}
+
+import scala.concurrent.duration._
+
+class ServiceInfoSpec extends AsyncFlatSpec with ScalatestRouteTest with Matchers {
+  val actorRefFactory = system
+  implicit val ec = system.dispatcher
+  implicit val timeout: Timeout = 5.seconds
+
+  val workflowStoreActor = actorRefFactory.actorOf(Props(new MockWorkflowStoreActor()))
+
+  CromwellLanguages.initLanguages(LanguageConfiguration.AllLanguageEntries)
+
+  behavior of "ServiceInfo"
+
+  val expectedResponse = WesStatusInfoResponse(Map("CWL" -> Set("v1.0"), "WDL" -> Set("draft-2", "1.0", "biscayne")),
+    List("1.0"),
+    Set("ftp", "s3", "demo-dos", "gcs", "oss", "http"),
```

Comment:
Just a heads-up, filesystem `demo-dos` is being removed in PR #4456, and being replaced by `drs` :)

Reply:
One of us has a rebase in our future :P